src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -325,8 +345,8 @@
     </PropertyGroup>
 
     <MakeDir Directories="$([System.IO.Path]::GetDirectoryName('$(RepoConsoleLogFile)'));
-                          $(ArtifactsShippingPackagesDir);
-                          $(ArtifactsNonShippingPackagesDir);
+                          $(RepoArtifactsShippingPackagesDir);
+                          $(RepoArtifactsNonShippingPackagesDir);
```

Comment:
I think you can remove these two lines. They were necessary before but I don't think they are still required. They cause empty directories to be created in your PR:

![image](https://github.com/dotnet/installer/assets/7412651/becb3e35-00c5-4c98-9b9e-c3dbd36c009d)

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/a50512dc17ae245c0ac70d263c90b7c632961889